### ISSUE TITLE
[Feature]: plot(FIMSFrame(data_mile1))

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [NOAA-FIMS/FIMS#174]
### Is your feature request related to a problem? Please describe.

Quick figures of the data would be helpful to limit the amount of code that scientists have to write themselves. Please provide comments on the figure below if you would like current changes. Note that we can always change this later. Just looking for quick comments today.

### Describe the solution you would like.

A method for plotting the input data where users can use `plot(FIMSFrame(data_mile1))`. I have implemented {ggplot2} code that produces the following figure.

### Describe alternatives you have considered

NA

### Additional context

![plot](https://user-images.githubusercontent.com/4108564/173886939-f2f51c4f-cbdf-41f4-be7c-c2158d763210.png)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by iantaylor-NOAA** [comment]
I really like the idea of an easy way to plot the input data before modeling.
The index and landings panels seem fine. The age comp and weight-at-age panels don't seem as useful. Assigning each age a different color would help, but for those two panels I would prefer plots of date vs age with value represented by bubble size or color.

Also, are we allowing multiple fleets in milestone 1, and if so, should we account for that in the figure?

**Comment by kellijohnson-NOAA** [comment]
Thanks @iantaylor-NOAA for the reminder about multiple fleets. Should we use [{nmfspallete}](https://nmfs-fish-tools.github.io/nmfspalette/)?

**Comment by iantaylor-NOAA** [comment]
Diving into color palettes might be beyond your initial request for quick comments but I can't help myself. I like the {nmfspallete} colors in general, but they don't seem to be designed for the visual display of quantitative information (based on a quick look at the guide here: https://drive.google.com/drive/folders/1MPFBbkYAn_HcZlVmZWxfYbLsKJ8sVVjw).

The only palette that provides a good visual difference between colors is "regional web" which works great for up to 6 colors (for our 6 NMFS regions), but after that (e.g. `nmfspalette::display_nmfs_palette("regional web", 7)`), you get colors that are pretty similar to each other. 

However, worrying about the ideal color palettes seems like a long way away given that we don't have a working model yet, let alone one with multiple fleets, so I'm OK with whatever you want that we can always change later.

**Comment by jimianelli** [comment]
Agree w/ Ian re age-specific ones not super helpful. Re multiple fleets perhaps a new page and facet for that? Also wondering if NMFS palette includes a default theme (not a fan of ggplot defaults, personally prefer ggthemes:theme_few(). Also nit picking, but +expand_limits(y=0) might be useful for index plots? also lazy question, is FIMSframe defined somewhere?

**Comment by k-doering-NOAA** [comment]
Re where the definition of FIMSFrame() is, it is currently in a branch, but will be moved to main (hopefully soon!): https://github.com/NOAA-FIMS/FIMS/blob/2ca4d99f67dd16b104a8a96f2a10957e0aa76218/R/fimsframe.R . Calling `FIMSFrame(mydata)` creates an s4 object of class FIMSFrame, where the class definition is also defined in this same script.

I am also not a ggplot default theme fan, but on the other hand, the brilliance of doing this as ggplots is that the user could manipulate saved ggplot output and add on the theme they would like.

**Comment by kellijohnson-NOAA** [comment]
Thanks for the comments. I changed the color scheme to use `Set3` 

https://github.com/NOAA-FIMS/FIMS/blob/2ca4d99f67dd16b104a8a96f2a10957e0aa76218/R/fimsframe.R#L49

which should be good enough for right now and the start of the next milestone because it provided the most colors of any default theme I found without needing another package. This is **ALWAYS** changeable later. I also chose to NOT make panel specific plots because this is just the default plot for `FIMSFrame` and not `FIMSFrameAge`. So, in theory this figure would need to accommodate every single type of data we could ever think about including. Thanks again for the comments they helped me think about other plotting methods that should be made for the future and small changes to make to this one. 🥂
